backend/models.py

```diff
@@ -1,3 +1,66 @@
-# from django.db import models
+from django.db import models
 
 # Create your models here.
+
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=150)
+
+
+class FieldOfStudy(models.Model):
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
+    name = models.CharField(max_length=150)
+    degree = models.IntegerField()
+
+
+class Yearbook(models.Model):
```

Comment:
Tu myślę, że trzeba będzie zapytać, czy jest coś takiego jak próg minimalny, czy on wynika tylko ze zgłoszeń i wyników

Reply:
racja, ale to póki co dopiszę go, bo ma w sumie sens